.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Closes ISSUE_NUMBER
```

Comment:
Looks good to me, this is the same template we approved before.

However, you improved some parts @Tommo-L.
I like the "// If need the help of test team, please describe the test scenario."

On the other hand, the requirement to open an Issue is not so good to me. This can be optional and indicated as a good practice.